.eleventy.js

```diff
@@ -61,9 +62,12 @@ module.exports = function(eleventyConfig) {
         return heroGen(""+id)
     })
 
+    eleventyConfig.addFilter("toAbsoluteUrl", function(url) {
+        return new URL(url, site.baseURL).href;
+    })
+
```

Comment:
Where is this filter used?

Reply:
https://github.com/flowforge/website/pull/370/files#diff-2c3c931c99a0419e176c109e0045c4080281011ec8beac012b5c1462b064abe5R6

Reply:
Added this to more places to DRY our code.